src/main/java/Deadline.java

```diff
@@ -0,0 +1,20 @@
+public class Deadline extends Task {
+
+    protected String deadline;
+
+    public Deadline(String name, String deadline) {
+        super(name);
+        this.deadline = deadline;
+    }
+
+    @Override
+    public String getTaskType() {
+        return "D";
+    }
+
+    @Override
+    public String toString() {
+        String[] strArr = deadline.split(" ", 2);
+        return name + " (" + strArr[0] + ":" + " " + strArr[1] + ")";
```

Comment:
You may wish to leave a line break or use `String::format` to improve readability:

```java
return String.format(
    "%s (: %s)",
    name,
    strArr[0],
    strArr[1]
);
```

Also, `strArr` should use more declarative naming in the plural form. `tokens` would be a better variable name.